packages/opc-timeline/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@one-platform/opc-timeline",
+  "version": "0.0.1",
```

Comment:
```suggestion
  "version": "0.0.1-prerelease",
```

Reply:
@gisumit I have added this to the opc-timeline thanks, should we add this by default in the generator?

Reply:
nope keep it as it is.